### DISCUSSION TITLE
ChatGPT變懶了？拒絕轉檔、表格亂畫，OpenAI坦言此事並承諾改善

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1864] [General]
文章中提及GPT4.0變懶惰了，不再確切去執行用戶所下達的指令，
像是ChatGPT說檔案太大無法轉檔、拒絕讀取連結等，
看完這篇文章後，想法是ChatGPT怎麼有點可愛，很像人類職業倦怠的樣子xd

不過想問有人有升級成GPT4.0嗎？～
我個人是沒有買，因為我用3.5就足夠了，是有怎樣的需求才會去做升級呢～

資料來源：https://www.thenewslens.com/article/197114

**Comment by vchien03** [comment]
好神奇XD，很想知道為甚麼會出現「懶惰」的狀況出現~
我是沒有買4.0，其實平常很少使用ChatGPT，不過之前在學程式語言時，用ChatGPT真的很有幫助！

**Comment by vchien03** [comment]
這讓我想到之前害怕未來世界會被AI取代的議題，難道AI開始有自我意識了嗎？

**Comment by TobywithDino** [comment]
職業倦怠聽起來好可愛XD 感覺4.0是一個很厲害得工具欸 不捨得花錢的話大家可以看看微軟最近推出的AI助理copilot，似乎有免費可以使用的chatGPT4.0

**Comment by chuanmin0317** [comment]
最近google有推出他們的新AI，叫做Gemini，聽說有媲美GPT4的能力，不過對中文的支援度還有點差，不過英文很好的話還是可以試試看的，重要的是現在是免費的，不像是GPT4要收費

**Comment by yubo0oo** [comment]
我覺得，文章中提到的GPT-4.0變懶惰這件事真的挺有趣的。看到ChatGPT說檔案太大無法轉檔、拒絕讀取連結，感覺它真的有點像人類在工作中的職業倦怠，這種擬人化的特質讓它顯得有點可愛。

至於升級成GPT-4.0，我個人也沒有買，因為我覺得使用3.5版本已經足夠滿足我的需求了。除非有特別高的需求，比如需要更高精度的回答、更多的功能支持或者進行更複雜的任務，才會考慮升級。我覺得大部分日常使用3.5版本就已經很不錯了，升級到4.0可能對專業用戶或者需要特定功能的人更有吸引力。

大家有升級成GPT-4.0嗎？使用體驗如何？有什麼特別的需求會讓你們選擇升級呢？歡迎大家分享自己的經驗和想法！
